app/models/user.rb

```diff
@@ -1,8 +1,15 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
   has_many :posts, foreign_key: 'author_id'
   has_many :likes, foreign_key: 'author_id'
   has_many :comments, foreign_key: 'author_id'
 
+  validates :email, presence: true, allow_blank: false
+  validates :encrypted_password, presence: true, allow_blank: false
```

Comment:
**OPTIONAL** - Good work done so far, but you do not need to add these validations as they are already handled by devise. Please fix this 😁